tests/integration/seccomp-notify.bats

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	teardown_seccompagent
```

Comment:
This should not be necessary -- bats is reliably calling teardown.

Reply:
Right, removed! When we opened the PR the teardown was usually called in other functions in the code, I think this is why this was here in the first place. But this has changed in runc since then :)

Reply:
Yes, this was cleaned up some time ago :)